main.js

```diff
@@ -7,73 +7,73 @@ class BookCollection {
     this.bookstore = JSON.parse(localStorage.getItem(this.BOOKS)) || [];
     this.myTites = document.getElementById('inputone');
     this.myAuth = document.getElementById('inputtwo');
-    this.myAdd = document.getElementById('butn');
-    this.myList = document.getElementById('list-of-books');
+    this.ourAdd = document.getElementById('butn');
+    this.List = document.getElementById('list-of-books');
   }
 
-  saveBooks() {
-    localStorage.setItem(this.BOOKS, JSON.stringify(this.bookstore ));
+  saveBookf() {
+    localStorage.setItem(this.BOOKS, JSON.stringify(this.bookstore));
   }
 
-  addBook() {
+  addBuks() {
     const title = this.myTites.value.trim();
-    const author = this.myAuth .value.trim();
+    const author = this.myAuth.value.trim();
 
     if (title && author) {
       const book = { title, author };
-      this.bookstore .push(book);
```

Comment:
- [ ] Optional: 
- I would advise you to use `concat` instead of `push` method because `push()` induces changes in the original array, whereas `concat()` makes no change in existing array values.